### DISCUSSION TITLE
How do I register for the weekly project meetings?

### DISCUSSION THREAD

**Discussion by arithran** [serverlessworkflow/specification#766] [General] [answered]
The Register link appears to be broken. 
<img width="868" alt="Screenshot 2023-05-25 at 10 26 39 PM" src="https://github.com/serverlessworkflow/specification/assets/1830088/b8a62605-86f7-423d-a85e-79da716bb577">

I'd like to join the project meeting/office hours to learn more about synapse 

How do I register for the weekly project meetings?

**Comment by ricardozanini** [comment]
Hi @arithran! We are fixing this link. The weekly meetings are suspended this upcoming week while we do a few internal housekeeping. Thank you! Please stay tuned! We will announce the changes soon in the CNCF Slack #serverlessworkflow channel.

**Comment by ricardozanini** [comment, marked as answer]
Hi! I forgot to post the update; you can see here how: https://github.com/serverlessworkflow/specification/discussions/799

There's no formal "registration" anymore. Just be there at the time described in the calendar. 👍
